daq/local_state_collector.py

```diff
@@ -38,6 +43,18 @@ def get_process_state(self, extended=True):
 
         return self._process_state
 
+    def _get_target_processes(self):
+        """Get target processes"""
+        procs = {}
+        for proc in psutil.process_iter():
+            for target_name, (target_cmd, target_regex) in self._target_procs.items():
+                if proc.name() == target_cmd:
+                    cmd_line_str = ''.join(proc.cmdline())
```

Comment:
double-checking -- are you sure this shouldn't be join-on-space rather than join-on-nothing?

Reply:
My intention was joining-on-nothing. Since the joined string is only used to filter out the processes by the regex, I suppose we don't need a space between each two command line fields.

Reply:
Got it. I wasn't sure if this was what was returned in the API itself. It's not, so we're god to go.